chain-impl-mockchain/src/transaction/transfer.rs

```diff
@@ -189,6 +198,12 @@ pub struct Output<Address> {
     pub value: Value,
 }
 
+impl<Address: Readable> Output<Address> {
+    pub fn from_address_and_value(address: Address, value: Value) -> Self {
```

Comment:
Actually, looking at the other constructors, `_and_value` seems out of place here, as they all have the second `Value` parameter. So it can be named `from_address` for the perfect feng shui.